### ISSUE TITLE
test ultra rapid

### ISSUE THREAD

**Issue by LucaFibbi** [XiaoGongWei/MG_APP#23]
Thank you, for last commit that it has fixed the abort.
I try at use ultra rapid orbits and clocks for an PPP solution.
The output position is almost always zero.
Is it the quality of the orbits and clocks that determines the result?

I attached the test case and here are show the options used:
TropDeal=UNB3m
Method=Kalman
Cut-off Angle=0
Kinematic No
Satellite system GREC
PPP NoSmooth
PPP Back No
run PPP
[test_ultrarapid.zip](https://github.com/XiaoGongWei/MG_APP/files/4567965/test_ultrarapid.zip)

**Comment by XiaoGongWei** [comment]
I have seen the product file, and the observation data is January 30, 2020, but the product lacks the whu20905 data.You can try again.

**Comment by LucaFibbi** [comment]
I wanted to test the use in real-time or quasi real-time, when
orbit and clock do not exist for the next day.
The orbit and clock files end at same epoch of observation file and the
algorithm for interpolation of satellite position and clock should work
also in this case. The calculation of zero position occurs from
the beginning of the period when there are all data of orbit and clock,
even if expected.
Is it the accuracy of these data that prevents the calculation of the position ?

**Comment by XiaoGongWei** [comment]
Yes, the accuracy of these data that prevents the calculation of the position. Because the accuracy of the external interpolation of orbital and clock is too low. You can't take an external interpolation between orbit and clock. External interpolationis not reasonable.

You can externally predict the orbital and clock, and save them in sp3 and clk formats. Then, turn off filter residual detection, and  use MG-APP processing observations by you predict products.
